deployment/tasks/fixtures.cap

```diff
@@ -1,9 +1,27 @@
 namespace :symfony do
     namespace :mongo do
-        desc 'Load fixtures'
-        task :load_fixtures do
-            on roles(:app) do
-                execute "cd #{fetch(:deploy_to)}/current && php app/console doctrine:mongodb:fixtures:load --env=prod "
+        namespace :all_fixtures do
```

Comment:
I think it would be more concise if the command was mongo:all:load instead of mongo:all_fixture:load and the same for others tasks

But as you wish